src/node/chainstate.cpp

```diff
@@ -168,15 +168,16 @@ ChainstateLoadResult LoadChainstate(ChainstateManager& chainman, const CacheSize
     Chainstate& validated_cs{chainman.InitializeChainstate(options.mempool)};
 
     // Load a chain created from a UTXO snapshot, if any exist.
-    bool has_snapshot = chainman.DetectSnapshotChainstate();
+    Chainstate* assumed_valid_cs{chainman.DetectAssumedValidChainstate()};
 
-    if (has_snapshot && options.wipe_chainstate_db) {
+    if (assumed_valid_cs && options.wipe_chainstate_db) {
```

Comment:
In commit 9c0ecea4c4df01f8ed3492e9a4f538fcef1e51df

This new `assumed_valid` terminology seems confusing to me, because assume valid and assumeutxo have different semantics and different activation heights. Snapshot seemed clear to me tbh.